.github/workflows/main.yaml

```diff
@@ -93,10 +93,14 @@ jobs:
 
     # Make the OS as close as possible to a standard one.
     # But make sure we have the required dev libraries.
-    - name: Prepare OS
+    - name: Remove Homebrew
```

Comment:
My goal was to have all the OS preparate task in a step... there should a balance between putting all tasks in a step, or creating a separate step for each executed command.

I prefer to have multiple steps, each step grouping multiple command that have the same purpose.

Reply:
ACK.